modules/installation-osp-fixing-subnet.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * installing/installing_openstack/installing-openstack-installer-user.adoc
+//
+//YOU MUST SET AN IFEVAL FOR EACH NEW MODULE
+
+[id="installation-osp-fixing-subnet_{context}"]
+= Setting a custom subnet for machines
+
+The IP range that the installation program chooses may not match the Neutron subnet that you create when installing {product-title} on your own infrastructure. If necessary, update the CIDR value for new machines by editing the installation configuration file.
+
+.Prerequisites
+
+* You have `install-config.yaml` as generated by the {product-title} installation program
+
+.Procedure
+
+. On a command line, browse to the directory that contains `install-config.yaml`.
+. From that directory, enter the command:
++
+----
+$ python -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.safe_load(open(path));
+data["networking"]["machineNetwork"][0]["cidr"] = "<subnet range>"; <1>
```

Comment:
you might want to add a comment prefix; like:

```Python
data["networking"]["machineNetwork"][0]["cidr"] = "<subnet range>"; # <1>
```

so that a careless copy-paster will still get valid code

Reply:
Neat suggestion. I'll see how this renders in d.o.c. and the portal. Thanks!

Reply:
Doesn't render, I'm afraid. :(